### ISSUE TITLE
Version 3.2 not starting due to workspace error

### ISSUE THREAD

**Issue by jkjkjkjkjk** [OpenRefine/OpenRefine#2150]
**Describe the bug**
Not starting due to workspace(?) error

**To Reproduce**
1. Windows kit unzipped
2. refine.bat started

**Current Results**
G:\programy\openrefine-2.7>"C:\Program Files\Java\jre1.8.0_221\bin\java.exe" -cp "server\classes;server\target\lib\*"  -
Drefine.data_dir=C:\Users\user\AppData\Roaming\OpenRefine -Xms1400M -Xmx1400M -Drefine.memory=1400M -Drefine.max_form_co
ntent_size=1048576 -Drefine.port=3333 -Drefine.host=127.0.0.1 -Drefine.webapp=main\webapp -Djava.library.path=server\tar
get\lib/native/windows com.google.refine.Refine
22:20:17.546 [            refine_server] Starting Server bound to '127.0.0.1:3333' (0ms)
22:20:17.548 [            refine_server] refine.memory size: 1400M JVM Max heap: 1407188992 (2ms)
22:20:17.556 [            refine_server] Initializing context: '/' from 'G:\programy\openrefine-2.7\webapp' (8ms)
22:20:18.173 [                   refine] Starting OpenRefine 3.2 [55c921b]... (617ms)
22:20:18.174 [                   refine] initializing FileProjectManager with dir (1ms)
22:20:18.174 [                   refine] C:\Users\user\AppData\Roaming\OpenRefine (0ms)
Exception in thread "main" java.lang.LinkageError: loader constraint violation: when resolving overridden method "com.go
ogle.refine.pcaxis.PCAxisImporter.createParserUIInitializationData(Lcom/google/refine/importing/ImportingJob;Ljava/util/
List;Ljava/lang/String;)Lcom/fasterxml/jackson/databind/node/ObjectNode;" the class loader (instance of edu/mit/simile/b
utterfly/ButterflyClassLoader) of the current class, com/google/refine/pcaxis/PCAxisImporter, and its superclass loader
(instance of org/mortbay/jetty/webapp/WebAppClassLoader), have different Class objects for the type com/fasterxml/jackso
n/databind/node/ObjectNode used in the signature
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Unknown Source)
        at java.lang.Class.privateGetPublicMethods(Unknown Source)
        at java.lang.Class.getMethods(Unknown Source)
        at org.mozilla.javascript.JavaMembers.discoverAccessibleMethods(JavaMembers.java:351)
        at org.mozilla.javascript.JavaMembers.discoverAccessibleMethods(JavaMembers.java:302)
        at org.mozilla.javascript.JavaMembers.reflect(JavaMembers.java:424)
        at org.mozilla.javascript.JavaMembers.<init>(JavaMembers.java:43)
        at org.mozilla.javascript.JavaMembers.lookupClass(JavaMembers.java:812)
        at org.mozilla.javascript.NativeJavaClass.initMembers(NativeJavaClass.java:49)
        at org.mozilla.javascript.NativeJavaObject.<init>(NativeJavaObject.java:44)
        at org.mozilla.javascript.NativeJavaClass.<init>(NativeJavaClass.java:43)
        at org.mozilla.javascript.NativeJavaClass.<init>(NativeJavaClass.java:39)
        at org.mozilla.javascript.WrapFactory.wrapJavaClass(WrapFactory.java:139)
        at org.mozilla.javascript.NativeJavaPackage.getPkgProperty(NativeJavaPackage.java:135)
        at org.mozilla.javascript.NativeJavaPackage.get(NativeJavaPackage.java:84)
        at org.mozilla.javascript.ScriptableObject.getProperty(ScriptableObject.java:2313)
        at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1579)
        at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1572)
        at org.mozilla.javascript.gen.file__G__programy_openrefine_2_7_webapp_extensions_pc_axis_module_MOD_INF_controll
er_js_10._c_init_1(file:/G:/programy/openrefine-2.7/webapp/extensions/pc-axis/module/MOD-INF/controller.js:39)
        at org.mozilla.javascript.gen.file__G__programy_openrefine_2_7_webapp_extensions_pc_axis_module_MOD_INF_controll
er_js_10.call(file:/G:/programy/openrefine-2.7/webapp/extensions/pc-axis/module/MOD-INF/controller.js)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:405)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3508)
        at org.mozilla.javascript.gen.file__G__programy_openrefine_2_7_webapp_extensions_pc_axis_module_MOD_INF_controll
er_js_10.call(file:/G:/programy/openrefine-2.7/webapp/extensions/pc-axis/module/MOD-INF/controller.js)
        at edu.mit.simile.butterfly.ButterflyModuleImpl.scriptInit(ButterflyModuleImpl.java:636)
        at edu.mit.simile.butterfly.ButterflyModuleImpl.init(ButterflyModuleImpl.java:94)
        at edu.mit.simile.butterfly.Butterfly.initializeModule(Butterfly.java:476)
        at edu.mit.simile.butterfly.Butterfly.configure(Butterfly.java:451)
        at edu.mit.simile.butterfly.Butterfly.init(Butterfly.java:308)
        at org.mortbay.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:440)
        at org.mortbay.jetty.servlet.ServletHolder.doStart(ServletHolder.java:263)
        at com.google.refine.RefineServer.configure(Refine.java:291)
        at com.google.refine.RefineServer.init(Refine.java:203)
        at com.google.refine.Refine.init(Refine.java:109)
        at com.google.refine.Refine.main(Refine.java:103)

**Expected behavior**
A clear and concise description of what you expected to happen or to show.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser Version: N/A
 - JRE or JDK Version: java version "1.8.0_221"

**OpenRefine (please complete the following information):**
 - OpenRefine 3.2

**Additional context**
After reading https://github.com/OpenRefine/OpenRefine/issues/1939 moved old data from C:\Users\user\AppData\Roaming\OpenRefine, but still have error.  Despite directory name "openrefine-2.7" it is really 3.2.

**Comment by wetneb** [comment]
Thanks for this report! Can you check if you have installed any extensions yourself? It is likely that this is caused by an incompatible extension.

**Comment by jkjkjkjkjk** [comment]
I don't exactly remember, but I found some at webapp\extensions. Tried to delete/rename these one by one, but still same error.
Finally, I tried clean install into new directory and everything works OK (from new directory), even with old projects. I compared  webapp\extensions in "old" and "new" installation, same directories but different filesize, so maybe there was something in these extensions causing the error.
